docs/products/opensearch/howto/migrate-snapshot-data-opensearch.md

```diff
@@ -400,7 +400,24 @@ The response shows the snapshot status and details if a snapshot is in progress:
 
 The `details` section is included only if a snapshot is in progress.
 
-## Verify the migration
+## Finalize the migration process
+
+After the restoration process is complete, Aiven for OpenSearch automatically deletes
+the snapshot repository used during the migration to clean up resources.
+
+### Reapply ISM policies and security configurations
+
+Next, reapply any necessary configurations:
+
+- **Reapply ISM policies**: Reapply any Index State Management (ISM) policies to the
+  restored indices. For more information, see
+  [Reapply ISM policies after snapshot restore](/docs/products/opensearch/howto/migrate-ism-policies).
+
+- **Update security configurations**: Review and reconfigure any security settings,
+  including OpenDistro security configurations. For more details, see
+  [Migration Opendistro security configuration](/docs/products/opensearch/howto/migrate-opendistro-security-config-aiven).
```

Comment:
Nitpicking to remove some potentially redundant word:

```suggestion
Reapply necessary configurations:

- **Reapply ISM policies**: Reapply Index State Management (ISM) policies to the
  restored indices. For more information, see
  [Reapply ISM policies after snapshot restore](/docs/products/opensearch/howto/migrate-ism-policies).

- **Update security configurations**: Review and reconfigure security settings,
  including OpenDistro security configurations. For more details, see
  [Migration Opendistro security configuration](/docs/products/opensearch/howto/migrate-opendistro-security-config-aiven).
```

Reply:
Thanks! Addressed them!